core/src/main/scala/org/bykn/bosatsu/Package.scala

```diff
@@ -332,12 +332,7 @@ object Package {
     def errorFor(t: (Type.Const, Exp, Type)): List[PackageError] =
       exportedTE.toDefinedType(t._1) match {
         case None =>
-          if (Type.TyConst(t._1) != Type.FnType)
```

Comment:
maybe we should put this back and remove the external struct stuff... it's better to programmatically define some of these types (Fn and Tuple if we stop the cons-cell encoding of Tuple to match Fn)